tests/integration/__tests__/with-parser-inference.js

```diff
@@ -28,6 +28,18 @@ describe("infers parser from filename", () => {
     ).toEqual("{}\n");
   });
 
+  test("json from .stylelintrc", () => {
+    expect(
```

Comment:
We do format test in `tests/format` dir, see https://github.com/prettier/prettier/pull/8105/files

Reply:
Isn't this suite for testing infer file format from filename with not extensions?

Reply:
Okay, works too.